### PR TITLE
Cluster tests

### PR DESCRIPTION
**Pull request by cgilmour**
This PR is for the romana end-to-end tests that are executed on the cluster.
Test scripts live in `test` with specific folders for the stack type (currently only `kubernetes-cluster`.

The setup script has a new `test` subcomand to install and run the tests for a specific stack type.
Each host produces some logs. At the end of the test, these logs are merged together to a single point and that point can be pushed up to..whoever is interested.

This is is the area where more test types/categories/scripts would be added.
Generally, each test should
* print a summary of whether it succeeded/failed
* exit with zero (success) or non-zero (failure)
* support a -v option for verbose mode
* be executable, with an appropriate shebang and such. Doesn't have to be bash, but it's often the easiest.
* clean up after itself, if possible

The ansible portion invokes certain "categories" of tests. These should be incremental, checking basic functionality before proceeding to complex interactions.

### CODE REVIEW

File: romana-install/tests.devstack.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Tests not developed yet
+- hosts: localhos
```
Comment (jbrendel) [inline romana-install/tests.devstack.yml]: "localhost"

File: romana-install/romana-setup [hunk 2]
```diff
@@ -25,7 +25,7 @@ usage() {
 	echo "Platforms:   aws (default), vagrant"
 	echo "Distro:      ubuntu (default), centos"
 	echo "Stack Types: devstack (default), kubernetes"
-	echo "Actions:     install (default), uninstall"
+	echo "Actions:     install (default), uninstall, test"
```
Comment (jbrendel) [inline romana-install/romana-setup]: So, "test" assumes that a cluster has been successfully installed and then runs whatever tests we have against the existing cluster?

Could you add an example line for that, please?

**Comment by jbrendel** [comment]
I assume that test scripts could also be written in Python?

File: romana-install/romana-setup (hunk 2 above)
Comment (cgilmour) [inline romana-install/romana-setup]: Example added in latest commit.

**Comment by cgilmour** [comment]
Correct, test scripts could be written in Python as long as they play by the "rules" mentioned in the initial PR comment.

**Comment by cgilmour** [comment]
One other thing: the way tests are configured and launched will probably need rewriting as the amount and complexity of the tests grows, but I expect the `test` subcommand and the "rules" for each executable to basically remain the same.
This PR is to get us started with _something_ and it can grow and be refactored from there.

**Comment by jbrendel** [comment]
Ok

**Pull request merged**